app/views/describeYourItem.scala.html

```diff
@@ -0,0 +1,64 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import uk.gov.hmrc.play.views.html._
+@import controllers.routes._
+@import models.{Mode, DescribeYourItem}
+@import views.ViewUtils._
+
+@(appConfig: FrontendAppConfig, form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = s"${errorPrefix(form)} ${messages("describeYourItem.title")}",
+    appConfig = appConfig,
+    bodyClasses = None) {
+
+    @helpers.form(action = DescribeYourItemController.onSubmit(mode), 'autoComplete -> "off") {
+
+        @components.back_link()
+
+        @components.error_summary(form.errors)
+
+        @components.heading("describeYourItem.heading")
+
+        <div class="panel panel-border-wide">
+          <p>Include as much information as you can, this will prevent any delays to your application.</p>
+        </div>
+
+        @components.input_text(
+            field = form("field1"),
+            label = messages("describeYourItem.field1")
+        )
+
+        <p>This may include:</p>
+
+        <ul class="list list-bullet">
+            <li>what it is used for</li>
+            <li>how it works</li>
+            <li>what it is made from</li>
+            <li>how it is packaged</li>
```

Comment:
Do these list items not need to be in `@messages`?

Reply:
I don't think they 'need' to be in messages seeing as this service is not going to be translated (as far as I'm told), but, yes like all other content I've added, I can move them to messages. I must have overlooked these.

Reply:
If Translation is not a requirement then thats fine :)

Reply:
It's just another 'its easier to do it as we go along' in terms of having everything in `messages`